### PR TITLE
fix(README): fix Camunda properties docs

### LINKED ISSUE #1001: Camunda Property Integration
### Context

I recently tried integrating the Camunda properties into the modeler/properties panel. I followed the README of this repo, where it says to integrate the CamundaExtensionModule as follows:

```
// use Camunda BPMN Moddle extension
import CamundaExtensionModule from 'camunda-bpmn-moddle/lib';
```

However, this does not work as the CamundaExtensionModule seems to be moved to the [behaviors repo](https://github.com/camunda/camunda-bpmn-js-behaviors) as [this post](https://forum.camunda.io/t/camunda-bpmn-moddle-lib-gone-in-breaking-changes-what-is-the-preferred-method-now/39908) / the [camunda-bpmn-moddle changelog](https://github.com/camunda/camunda-bpmn-moddle/blob/main/CHANGELOG.md) suggests. 

I am now integrating Camunda properties like this:
```
import { BpmnPropertiesPanelModule, BpmnPropertiesProviderModule, CamundaPlatformPropertiesProviderModule } from "bpmn-js-properties-panel";
import camundaPlatformBehaviors from "camunda-bpmn-js-behaviors/lib/camunda-platform";
const camundaModdleDescriptor = require("camunda-bpmn-moddle/resources/camunda.json");

return new BpmnModeler({
            additionalModules: [
                BpmnPropertiesPanelModule,
                BpmnPropertiesProviderModule,
                CamundaPlatformPropertiesProviderModule,
                camundaPlatformBehaviors
            ],
            moddleExtensions: {
                camunda: camundaModdleDescriptor
            }
        });
```

This seems to work fine so i think it is the latest and actual way to integrate Camunda properties into the properties panel? It would be nice if you could update the readme if that is the case.

## Why should we do it?

To keep the documentation in line with the updates and help users to get the modeler running. 

## General questions
I was a little unsure what the `CamundaPlatformPropertiesProviderModule`, `camundaPlatformBehaviors` and `camundaModdleDescriptor` are really doing. As far as i understand, the ModdleDescriptor is just a list of all possible extensions and the PropertiesProviderModule depends on the corresponding moddleDescriptor to provide all the right extensions for the properties panel as the behaviors ensure the right extensions on the right modeling elements and adds this kind of "business logic" to the properties. I know, as a user i do not really have to know about the internals and how these things are connected, but i am still curios to get a little clarification on this. 🙂

Also, from a user point of view, i think it would be cool to have this functionality wrapped in only one lib. This way it would be a lot easier to get things running and updating dependencies, as the user only has to care about one lib and not about between-lib entanglements and compatibilities, especially when they provide only one feature in total like the Camunda property integration. E.g., provide one lib that includes all stuff for camunda 7 and can be integrated as an additionalModule into the modeler.

Thank You!

### PR DESCRIPTION
**Pull request by philippfromme**
Closes #1001

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,14 +104,13 @@ const modeler = new BpmnModeler({
   additionalModules: [
     BpmnPropertiesPanelModule,
     BpmnPropertiesProviderModule,
-    CamundaPlatformPropertiesProviderModule,
-    CamundaExtensionModule
+    ZeebePropertiesProviderModule,
+    camundaCloudBehaviors
```
Comment (nikku) [inline README.md:108]:
```suggestion
    ZeebeBehaviors
```

File: README.md [hunk 2]
```diff
@@ -76,26 +76,25 @@ propertiesPanel.attachTo('#other-properties');
...
 
-// use Camunda BPMN Moddle extension
-import CamundaExtensionModule from 'camunda-bpmn-moddle/lib';
+// Camunda 8 moddle extension
+import zeebeModdle from 'zeebe-bpmn-moddle/resources/zeebe';
 
-// use Camunda BPMN namespace
-import camundaModdleDescriptors from 'camunda-bpmn-moddle/resources/camunda';
+// Camunda 8 behaviors
+import camundaCloudBehaviors from 'camunda-bpmn-js-behaviors/lib/camunda-cloud';
```
Comment (nikku) [inline README.md:97]:
```suggestion
import ZeebeBehaviorsModule from 'camunda-bpmn-js-behaviors/lib/camunda-cloud';
```

**Review by nikku — COMMENTED** [review]
Looks good. I added two minor suggestions (let's remove `cloud` where we can).

**Comment by philippfromme** [comment]
> Looks good. I added two minor suggestions (let's remove `cloud` where we can).

Changed the naming. Those `cloud` and `platform` names are going to haunt us for a long time.

**Review by nikku — APPROVED** [review]

**Pull request merged**